modules/card.js

```diff
@@ -223,6 +230,9 @@ class MtgCardLoader {
             }
 
             let description = this.generateDescriptionText(card);
+            if (command.match(/^art/)) {
```

Comment:
I wonder if this should be refactored into `generateDescriptionText`

Reply:
I would have to change the method signature to add command as a parameter, but I think this is the only place the function is called, so it could be done.

Following the pattern of the other sub-commands, I would add the artist at the bottom, by adding the field to the embed, but I prefer the cleaner look of deleting the description.

Reply:
I would actually prefer it to be grouped with the other sub-commands, after the embed is generated, in line 274 (see below)